test/chemistry/test_mes_gsc_calculation.py

```diff
@@ -52,6 +52,13 @@ def test_npme(self):
         res = calc.solve(self.driver)
         self.assertAlmostEqual(res.energy, self.reference_energy, places=6)
 
```

Comment:
Maybe the whole test should be renamed to 'test_groundstatesolver` which would be more descriptive and match new naming better